package.json

```diff
@@ -8,6 +8,12 @@
   "module": "dist/downshift.esm.js",
   "typings": "typings/index.d.ts",
   "scripts": {
+    "start": "nps",
+    "test": "nps test",
+    "prerelease": "npm start validate",
+    "release": "npm publish && npm dist-tag add downshift@$npm_package_version rc",
+    "precommit": "lint-staged && opt --in pre-commit --exec \"npm start validate\"",
+    "flow": "flow"
```

Comment:
,

Reply:
missed comma I mean

Reply:
Shouldn't `flow` be added to the `validate` script? I'm not sure if that can be done via `kcd-scripts` or if it has to be a separate command in that script. @kentcdodds, thoughts?